index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="./styles.css">
+</head>
+
+<body>
+    <header>
+        <nav>
+            <a href="./index.html" class="logo">My Logo</a>
+            <div class="menu">
+                <div></div>
+                <div></div>
+                <div></div>
+            </div>
+        </nav>
+    </header>
+    <section class="main-section">
+        <h1>I'm David <br>Glad to see you!</h1>
+        <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work
+            and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+        </p>
+        <p class="lc">LET'S CONNECT</p>
+        <div class="icons">
+            <a href="./index.html"><img src="./images/Vector.svg" alt="twitter icon"></a>
+            <a href="./index.html"><img src="./images/Linkedin icon.svg" alt="Linkedin icon"></a>
+            <a href="./index.html"><img src="./images/Vector (1).svg" alt=""></a>
+            <a href="./index.html"><img src="./images/Vector (2).svg" alt="github icon"></a>
+            <a href="./index.html"><img src="./images/Vector (3).svg" alt=""></a>
+        </div>
```

Comment:
- [ ] The list of social links is well designed, but it would be better wrapped with a `ul` tag with the `li` tags around each `a` tag for better semantics. Kindly change that.